app/bankaccountverification/web/business/BusinessAccountExistsIndeterminateViewModel.scala

```diff
@@ -25,7 +25,8 @@ case class BusinessAccountExistsIndeterminateViewModel(journeyId: String,
                                                        session: BusinessAccountDetails,
                                                        serviceIdentifier: String,
                                                        continueUrl: String,
-                                                       welshTranslationsAvailable: Boolean) {
+                                                       welshTranslationsAvailable: Boolean,
+                                                       journey: Option[Boolean]) {
```

Comment:
Don't think we need this? Or if we do, should it be called:
`useNewGovUkServiceNavigation`